### PR TITLE
CASMCMS-9112: Added two SBPS parameters

### PR DESCRIPTION
**Pull request by jsollom-hpe**
Added information about two parameters that are needed for the Scalable Boot Projection Service (SBPS).

# Description

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/iuf/workflows/configuration.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
 
 1. Edit the `compute-and-uan-bootprep.yaml` and `management-bootprep.yaml` files to account for any site deviations from the default values. For example:
     - Comment out the `slurm-site` CFS configuration layer and uncomment the `pbs-site` CFS configuration layer in `compute-and-uan-bootprep.yaml` if PBS is the preferred workload manager
+    - Comment out the SBPS `rootfs_provider` and `rootfs_provider_passthrough` parameters and uncomment the CPS `rootfs_provider` and `rootfs_provider_passthrough` parameters, if CPS is the preferred method to project content.
```
Comment (don-bahls-hpe) [inline operations/iuf/workflows/configuration.md]: if DVS with CPS is the preferred method to project content?

File: operations/iuf/workflows/configuration.md [hunk 2]
```diff
@@ -59,7 +60,17 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
    There are comments at the top of the `product_vars.yaml` file that describe the variables and related details. The following are a few examples of `site_vars.yaml` changes:
     - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
-
+    - Add a 'system' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
```
Comment (don-bahls-hpe) [inline operations/iuf/workflows/configuration.md]: I think you want it to be `system-name` to match the value below.

Comment (jsollom-hpe) [reply operations/iuf/workflows/configuration.md]: Fixed.

File: operations/iuf/workflows/configuration.md (hunk 1 above)
Comment (jsollom-hpe) [reply operations/iuf/workflows/configuration.md]: Changed.

File: operations/iuf/workflows/configuration.md [hunk 3]
```diff
@@ -78,6 +89,8 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
        default:
          network_type: "cassini"
          suffix: "-test01"
+	 system-name: "my-system"
```
Comment (don-bahls-hpe) [inline operations/iuf/workflows/configuration.md]: The indentation on these two variables seems incorrect.   Or at least different than the values above

Comment (jsollom-hpe) [reply operations/iuf/workflows/configuration.md]: Emacs inserted tabs when I wanted spaces. Fixed.

**Review by don-bahls-hpe — APPROVED** [review]

**Review by don-bahls-hpe — APPROVED** [review]

File: operations/iuf/workflows/configuration.md [hunk 4]
```diff
@@ -97,6 +110,9 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
        ./site_vars.yaml
        ```
 
+Until Jira CRAYSAT-1900 is resolved, the sat tool will not automatically replace the variables `system-name` and `site-domain` with the values defined in site_vars.yaml in the ${ADMIN_DIR}/bootprep/compute-and-uan-bootprep.yaml file. Therefore, they must
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
Until Jira CRAYSAT-1900 is resolved, the sat tool will not automatically replace the variables `system-name` and `site-domain` with the values defined in `site_vars.yaml` in the `${ADMIN_DIR}/bootprep/compute-and-uan-bootprep.yaml` file. Therefore, they must
```

File: operations/iuf/workflows/configuration.md [hunk 5]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
+      - See the following page in the CSM documentation:  'operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'
+        - If the 'docs-csm' RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/'.
+        - Otherwise, it can be found under the appropriate release branch in <https://github.com/Cray-HPE/docs-csm>.
+      - This documentation indicates how to find the system-name.
+    - Add a 'site-domain' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this domain name as the second part of the domain name.  Do not add if not using SBPS.
+      - See the following page in the CSM documentation:  'operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'
+        - If the 'docs-csm' RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/'.
+        - Otherwise, it can be found under the appropriate release branch in <https://github.com/Cray-HPE/docs-csm>.
+      - This documentation indicates how to find the site-domain.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
      - This documentation indicates how to find the `site-domain`.
```

File: operations/iuf/workflows/configuration.md [hunk 6]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
     - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+    - Add a 'system-name' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
+      - See the following page in the CSM documentation:  'operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'
+        - If the 'docs-csm' RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/'.
+        - Otherwise, it can be found under the appropriate release branch in <https://github.com/Cray-HPE/docs-csm>.
+      - This documentation indicates how to find the system-name.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
      - This documentation indicates how to find the `system-name`.
```

File: operations/iuf/workflows/configuration.md [hunk 7]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
    There are comments at the top of the `product_vars.yaml` file that describe the variables and related details. The following are a few examples of `site_vars.yaml` changes:
     - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+    - Add a 'system-name' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
+      - See the following page in the CSM documentation:  'operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
      - See the following page in the CSM documentation:  `operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md`
```

File: operations/iuf/workflows/configuration.md [hunk 8]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
+        - If the 'docs-csm' RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/'.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
        - If the `docs-csm` RPM is installed on a node, then this page can be found under `/usr/share/doc/csm/`.
```

File: operations/iuf/workflows/configuration.md [hunk 9]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
+        - Otherwise, it can be found under the appropriate release branch in <https://github.com/Cray-HPE/docs-csm>.
+      - This documentation indicates how to find the system-name.
+    - Add a 'site-domain' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this domain name as the second part of the domain name.  Do not add if not using SBPS.
+      - See the following page in the CSM documentation:  'operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'
+        - If the 'docs-csm' RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/'.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
        - If the `docs-csm` RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/'.
```

File: .spelling [hunk 10]
```diff
@@ -1077,3 +1077,16 @@ enablement
...
+Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md
+docs_csm
+usr
+csm
+Jira
+CRAYSAT-1900
+site_vars.yaml
+ADMIN_DIR
+compute-and-uan-bootprep
+yaml
```
Comment (rustydb) [inline .spelling]:
```suggestion
- operations/iuf/workflows/configuration.md
Jira
CRAYSAT-1900
```

**Review by rustydb — CHANGES_REQUESTED** [review]
Use backticks to wrap computer terminal words or code rather than updating the `.spelling` file.

File: operations/iuf/workflows/configuration.md (hunk 7 above)
Comment (nrockershousen) [inline operations/iuf/workflows/configuration.md]: We should use a link to this section. Something like..

See the `[Section Title](path/to/section)` procedure for more information.

File: operations/iuf/workflows/configuration.md [hunk 11]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+    - Add a 'system-name' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
+      - See the following page in the CSM documentation:  'operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'
+        - If the 'docs-csm' RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/'.
+        - Otherwise, it can be found under the appropriate release branch in <https://github.com/Cray-HPE/docs-csm>.
+      - This documentation indicates how to find the system-name.
+    - Add a 'site-domain' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this domain name as the second part of the domain name.  Do not add if not using SBPS.
+      - See the following page in the CSM documentation:  'operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'
```
Comment (nrockershousen) [inline operations/iuf/workflows/configuration.md]: Same comment as above.

File: operations/iuf/workflows/configuration.md (hunk 7 above)
Comment (jsollom-hpe) [reply operations/iuf/workflows/configuration.md]: I fixed these up.

File: operations/iuf/workflows/configuration.md (hunk 11 above)
Comment (jsollom-hpe) [reply operations/iuf/workflows/configuration.md]: I fixed these up.

File: operations/iuf/workflows/configuration.md [hunk 12]
```diff
@@ -97,6 +110,9 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
        ./site_vars.yaml
        ```
 
+Until Jira CRAYSAT-1900 is resolved, the sat tool will not automatically replace the variables `system-name` and `site-domain` with the values defined in `site_vars.yaml` in the `${ADMIN_DIR}/bootprep/compute-and-uan-bootprep.yaml` file. Therefore, they must
```
Comment (nrockershousen) [inline operations/iuf/workflows/configuration.md]: Please remove this internal Jira reference. We should avoid this in the public docs.
Maybe start this line with..

SAT will not automatically replace..

Comment (jsollom-hpe) [reply operations/iuf/workflows/configuration.md]: Dirty laundry hidden. We do expect the fix to CRAYSAT-1900 imminently, so my plan was never to leave this comment in for long. It was mostly to reassure the installers.

**Review by nrockershousen — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

File: operations/iuf/workflows/configuration.md [hunk 13]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
     - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+    - Add a 'system-name' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
+      - See the procedure [Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS](../../../operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md#boot-set-rootfs_provider_passthrough-parameter) for more information.
+        - If the `docs-csm` RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'. See the `Boot set rootfs_provider_passthrough parameter` section for more details.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
        - If the `docs-csm` RPM is installed on a node, then this page can be found under `/usr/share/doc/csm/operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md`. See the 
          `Boot set rootfs_provider_passthrough parameter` section for more details.
```

File: operations/iuf/workflows/configuration.md [hunk 14]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
    There are comments at the top of the `product_vars.yaml` file that describe the variables and related details. The following are a few examples of `site_vars.yaml` changes:
     - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+    - Add a 'system-name' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
+      - See the procedure [Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS](../../../operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md#boot-set-rootfs_provider_passthrough-parameter) for more information.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
      - See the procedure [Create a Session Template to Boot Compute Nodes with SBPS](../../../operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md#boot-set-rootfs_provider_passthrough-parameter) for more information.
```

File: operations/iuf/workflows/configuration.md [hunk 15]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
+        - If the `docs-csm` RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'. See the `Boot set rootfs_provider_passthrough parameter` section for more details.
+        - Otherwise, it can be found under the appropriate release branch in <https://github.com/Cray-HPE/docs-csm>.
+      - This documentation indicates how to find the `system-name`.
+    - Add a 'site-domain' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this domain name as the second part of the domain name.  Do not add if not using SBPS.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
    - Add a `site-domain` entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this domain name as the second part of the domain name.  Do not add if not using SBPS.
```

File: operations/iuf/workflows/configuration.md [hunk 16]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
    There are comments at the top of the `product_vars.yaml` file that describe the variables and related details. The following are a few examples of `site_vars.yaml` changes:
     - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+    - Add a 'system-name' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
    - Add a `system-name` entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
```

File: operations/iuf/workflows/configuration.md [hunk 17]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
+      - See the procedure [Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS](../../../operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md#boot-set-rootfs_provider_passthrough-parameter) for more information.
+        - If the `docs-csm` RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'. See the `Boot set rootfs_provider_passthrough parameter` section for more details.
+        - Otherwise, it can be found under the appropriate release branch in <https://github.com/Cray-HPE/docs-csm>.
+      - This documentation indicates how to find the `system-name`.
+    - Add a 'site-domain' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this domain name as the second part of the domain name.  Do not add if not using SBPS.
+      - See the procedure [Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS](../../../operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md#boot-set-rootfs_provider_passthrough-parameter) for more information.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
      - See the procedure [Create a Session Template to Boot Compute Nodes with SBPS](../../../operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md#boot-set-rootfs_provider_passthrough-parameter) for more information.
```

File: operations/iuf/workflows/configuration.md [hunk 18]
```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
+        - If the `docs-csm` RPM is installed on a node, then this page can be found under '/usr/share/doc/csm/operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'. See the `Boot set rootfs_provider_passthrough parameter` section for more details.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
        - If the `docs-csm` RPM is installed on a node, then this page can be found under `/usr/share/doc/csm/operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md`. See the `Boot set rootfs_provider_passthrough parameter` section for more details.
```

File: operations/iuf/workflows/configuration.md [hunk 19]
```diff
@@ -97,6 +110,9 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
        ./site_vars.yaml
        ```
 
+SAT will not automatically replace the variables `system-name` and `site-domain` with the values defined in `site_vars.yaml` in the `${ADMIN_DIR}/bootprep/compute-and-uan-bootprep.yaml` file. Therefore, they must
+be replaced manually. Use the same directions listed above for finding the values for these variables to populate the site_vars.yaml file and use them in the ${ADMIN_DIR}/bootprep/compute-and-uan-bootprep.yaml file.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
be replaced manually. Use the same directions listed above for finding the values for these variables to populate the site_vars.yaml file and use them in the `${ADMIN_DIR}/bootprep/compute-and-uan-bootprep.yaml` file.
```

**Review by rustydb — CHANGES_REQUESTED** [review]

File: operations/iuf/workflows/configuration.md [hunk 20]
```diff
@@ -59,6 +60,17 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
     - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+    - Add a `system-name` entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
+      - See the procedure [Create a Session Template to Boot Compute Nodes with SBPS](../../../operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md#boot-set-rootfs_provider_passthrough-parameter) for more information.
+        - If the `docs-csm` RPM is installed on a node, then this page can be found under `/usr/share/doc/csm/operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md`. See the 
+          `Boot set rootfs_provider_passthrough parameter` section for more details.
```
Comment (rustydb) [inline operations/iuf/workflows/configuration.md]:
```suggestion
          "Boot set `rootfs_provider_passthrough` parameter" section for more details.
```

**Review by rustydb — APPROVED** [review]

**Pull request merged**